LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Pixelgen Technologies AB
+Copyright (c) The nf-core/pixelator team
```

Comment:
```suggestion
Copyright (c) Pixelgen Technologies AB
```
You can also ignore this one since that company paid to get this pipeline built. It's also something they should adapt themselves if they want